lib/config/fileconf_test.go

```diff
@@ -332,6 +334,34 @@ func TestAuthenticationSection(t *testing.T) {
 	}
 }
 
+func TestAuthenticationConfig_Parse_StaticToken(t *testing.T) {
+	roles := []string{"Auth", "Node", "Proxy"}
+	joinedRoles := strings.Join(roles[:], ",")
+	var token StaticToken = StaticToken(fmt.Sprintf("%s:token", joinedRoles))
+
+	provisionToken, err := token.Parse()
+	require.NoError(t, err)
+
+	require.Len(t, provisionToken.Roles, len(roles))
+	for i := range roles {
+		require.Equal(t, string(provisionToken.Roles[i]), roles[i])
+	}
+	require.Equal(t, provisionToken.Token, "token")
+
+	// test parsing a windows filepath which contains a colon
+	windowsTokenPath := "C:\\path\\to\\some\\token\\file"
```

Comment:
```suggestion
	windowsTokenPath := `C:\path\to\some\token\file`
```

Reply:
I appreciate the comments, as you can probably tell I'm still new to Go.